### ISSUE TITLE
Loss doesn't decrease and AUC doesn't improve

### ISSUE THREAD

**Issue by davidsonic** [khuangaf/PyTorch-Geometric-YooChoose#1]
I actually tried the buy-click notebook, with smaller dataset and evaluated on the train set itself. The loss however, doesn't decrease and AUC doesn't improve. Have you encountered this before, what could be possible reasons?

![image](https://user-images.githubusercontent.com/15846113/59469975-3d7ddb00-8deb-11e9-9047-992be8f41427.png)

**Comment by davidsonic** [comment]
Problem solved!
